campaigns/windows-10/logo-swap.html

```diff
@@ -0,0 +1,232 @@
+                       }
+                       clearInterval(check_default_timeout);
+                   }
+               }, function noDefault () {
+                   if (is_firefox_default !== false) {
+                       showNonDefaultContent();
+                       is_firefox_default = false;
+                   }
+               });
+           }, 1000);
+
+           function showDefaultContent () {
+               snippet_text_default.style.display = 'block';
+               snippet_text_success.style.display = 'none';
+               snippet_content_default.style.display = 'block';
+               snippet_content_not_default.style.display = 'none';
+               snippet_icon.src = '{{ icon }}';
+
+               for (var i=0; i < buttons.length; i++) {
+                   buttons[i].addEventListener('click', function (e) {
+                       e.preventDefault();
+                       sendMetric('conversion-is-default-click', function () {
+                           window.location = '{{ url_default|safe }}';
+                       });
+                   }, false);
+               }
+           }
+
+           function showSuccessContent () {
+               snippet_text_default.style.display = 'none';
+               snippet_text_success.style.display = 'block';
+               snippet_content_default.style.display = 'block';
+               snippet_content_not_default.style.display = 'none';
+               snippet_icon.src = '{{ icon }}';
+
+               for (var i=0; i < buttons.length; i++) {
+                   buttons[i].addEventListener('click', function (e) {
+                       e.preventDefault();
+                       sendMetric('conversion-is-default-click', function () {
+                           window.location = '{{ url_default|safe }}';
+                       });
+                   }, false);
+               }
+           }
+
+           function showNonDefaultContent () {
+               snippet_content_default.style.display = 'none';
+               snippet_content_not_default.style.display = 'block';
+
+               for (var i=0; i < buttons.length; i++) {
+                   buttons[i].addEventListener('click', function (e) {
+                       e.preventDefault();
+                       Mozilla.UITour.setConfiguration('defaultBrowser');
+                       sendMetric('conversion-is-not-default-click');
+                       cta_clicked = true;
+                   }, false);
+               }
+           }
+
+           function isDefault (yesDefault, noDefault) {
+               Mozilla.UITour.getConfiguration('appinfo', function(config) {
+                   if (config && config.defaultBrowser === true) {
+                       yesDefault();
+                   } else if (config && config.defaultBrowser === false) {
+                       noDefault();
+                   } else {
+                       yesDefault();
+                   }
```

Comment:
nit: This can be refactored to 
```
 if (config && config.defaultBrowser === false) {
noDefault();
                   } else {
                       yesDefault();
                   }
```